packages/wpcom.js/package.json

```diff
@@ -0,0 +1,48 @@
+{
+	"name": "wpcom",
+	"description": "Official JavaScript library for the WordPress.com REST API",
+	"version": "5.4.2",
```

Comment:
Isn't this technically a breaking change, since the paths are changing inside the package?

Reply:
It depends on whether you consider the paths inside the package part of the public API, or an internal implementation detail 🙂 

I think it's safe to do a major update to 6.0.0, even though I'm not 100% sure if it's really necessary.

Reply:
That's a good question; I assumed it was part of the public API, but I could see the argument being made for internal implementation detail. I'll leave it to your judgement. I assume we won't be releasing anything for now, and probably not before making other breaking changes?